rero_ils/query.py

```diff
@@ -203,11 +203,6 @@ def search_factory_for_all_interfaces(view, search):
             search = search.filter('term', organisation__pid=org['pid'])
         # masked records are hidden for all public interfaces
         search = search.filter('bool', must_not=[Q('term', _masked=True)])
-    # Logic for admin interface
```

Comment:
Can you change de function name that suggest to be used by all resources. Something like `search_factory_for_items_and_holdings`. Thanks.